modules/wallet.go

```diff
@@ -187,10 +187,9 @@ type (
 		Drop()
 	}
 
-	// Wallet stores and manages siacoins and siafunds. The wallet file is
-	// encrypted using a user-specified password. Common addresses are all
-	// dervied from a single address seed.
-	Wallet interface {
+	// EncryptionManager can encrypt, lock, unlock, and indicate the current
+	// status of the EncryptionManager.
+	EncryptionManager interface {
```

Comment:
The value is the same as splitting up a large function into smaller functions: it breaks the code into more edible - and potentially more easily tested - chunks.

Having a 25 function interface didn't seem great to me. Breaking it into 3 chunks is nicer.